packages/components/src/SubHeaderBar/InputTitleSubHeader/TitleSubHeader.component.js

```diff
@@ -5,7 +5,12 @@ import { Action } from '../../Actions';
 import theme from './InputTitleSubHeader.scss';
 import { getDefaultTranslate } from '../../translate';
 
-function TitleSubHeader({ title, subTitle, onEdit, t }) {
+function noop() {}
+
+function TitleSubHeader({ title, subTitle, onEdit, editable, t }) {
+	const wordingClassNames = editable
+	? classNames(theme['tc-subheader-details-text-title-wording-with-hover'], 'tc-subheader-details-text-title-wording-with-hover')
+	: classNames(theme['tc-subheader-details-text-title-wording'], 'tc-subheader-details-text-title-wording');
```

Comment:
Please use `classNames` properly to get rid of the ternary operation.

Reply:
Look at my review below (it is marked as outdated because I submited it without refreshing :( )